core/src/test/scala/org/apache/gearpump/cluster/scheduler/PrioritySchedulerSpec.scala

```diff
@@ -58,7 +59,7 @@ class PrioritySchedulerSpec(_system: ActorSystem) extends TestKit(_system) with
       scheduler.tell(RequestResource(appId, request1), mockAppMaster.ref)
       scheduler.tell(RequestResource(appId, request2), mockAppMaster.ref)
       scheduler.tell(ApplicationFinished(appId), mockAppMaster.ref)
-      scheduler.tell(WorkerRegistered(workerId1), mockWorker1.ref)
+      scheduler.tell(WorkerRegistered(workerId1, MasterInfo(null, 0)), mockWorker1.ref)
```

Comment:
Can we add a constant to represent MasterInfo(null, 0) instead of writing everywhere?